### PR TITLE
Create Dev_Doc_Conda_Forge.md

### PR DESCRIPTION
**Pull request by shreyneil**
This pull request is in reference with the following issue https://github.com/weecology/retriever/issues/975 .
Please verify and get back.

### CODE REVIEW

**Comment by henrykironde** [comment]
Did you build the docs locally to see the results?

**Comment by shreyneil** [comment]
I made it using atom and and previewed it there itself. 
Is there any other way you have in your mind @henrykironde ?

**Comment by henrykironde** [comment]
Hi @shreyneil, 
Move content to this file https://github.com/weecology/retriever/blob/master/docs/developer.rst.
The file is not .md but .rst(restructured text).

Add a good tittle and then add this content in `Dev_Doc_Conda_Forge.md`.

To build the docs locally, You will have to follow Update Documentation on developers guide
http://retriever.readthedocs.io/en/latest/developer.html

As you add these changes, could you also change from `update Documentation` to `Update Documentation`. i.e lower case `u` to upper case `U` 

Let me know if anything fails to go your way.

**Comment by shreyneil** [comment]
I built the docs locally.
All the changes are done as asked,  @henrykironde .

File: Dev_Doc_Conda_Forge.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
```
Comment (henrykironde) [inline Dev_Doc_Conda_Forge.md]: You forgot to remove this file

File: docs/developer.rst [hunk 2]
```diff
@@ -144,6 +144,25 @@ Pull requests submitted to the repository will automatically be tested using
...
 page.
 
+Creating or Updating a Conda Release
+====================================
+
+Fork the https://github.com/conda-forge/staged-recipes repository.
+    
+Make a new folder in recipes for your package and name it meta.yml as this 
+https://github.com/conda-forge/staged-recipes/pull/3307/files. 
+
```
Comment (zhangcandrew) [inline docs/developer.rst]: Change: 
```
Make a new folder in recipes for your package and name it meta.yml as this 
https://github.com/conda-forge/staged-recipes/pull/3307/files. 
```
to
```
Make a new folder for your package in the recipes folder and create a new file 
in it called meta.yml. 
See the retriever `example <https://github.com/conda-forge/staged-recipes/pull/3307/files>`_ .

File: docs/developer.rst [hunk 3]
```diff
@@ -144,6 +144,25 @@ Pull requests submitted to the repository will automatically be tested using
...
+You can Look at the example recipe, read the documentation or refer to the FAQs 
+section for more help.
+
```
Comment (zhangcandrew) [inline docs/developer.rst]: Change: 
```
You can Look at the example recipe, read the documentation or refer to the FAQs 
section for more help.
```
to
```
You can look at the example recipe, read the documentation, or refer to the FAQ 
section for more help.
```

File: docs/developer.rst [hunk 4]
```diff
@@ -144,6 +144,25 @@ Pull requests submitted to the repository will automatically be tested using
...
+section for more help.
+    
+Open a pull request. Building of your package will be tested on Windows, Mac and Linux.
+    
+When your pull request is merged a new repository, called a feedstock, will be created 
+in the github conda-forge organization, and build/upload of your package will automatically 
+be triggered. Once complete, the package is available on conda-forge.
+    
+You can update the file i.e. meta.yml if you want a change or update and open another pull 
+request which will act as an update to your current pacakage.
```
Comment (zhangcandrew) [inline docs/developer.rst]: Change: 
```
You can update the file i.e. meta.yml if you want a change or update and open another pull 
request which will act as an update to your current pacakage.
```
to
```
To make changes or updates to your current package, you should make another pull request to your meta.yml file.

**Comment by zhangcandrew** [comment]
Hey @shreyneil, I think it looks pretty good! I just made some minor changes for grammar and things like that. I didn't get to build these files locally yet, so I'm not sure if the hyperlinking works yet, but I'll get around to that in a little bit and update this comment later.

**Comment by zhangcandrew** [comment]
Just built it locally, the hyperlinking syntax I gave you should work!

**Comment by shreyneil** [comment]
Done, Please verify and get back.

File: docs/developer.rst [hunk 5]
```diff
@@ -144,7 +144,25 @@ Pull requests submitted to the repository will automatically be tested using
...
+section for more help.
+    
+Open a pull request. Your package will be tested on Windows, Mac and Linux.
+    
+When your pull request is merged, a new repository called a feedstock will be created 
+in the github conda-forge organization, and your package will automatically 
+be uploaded and begin to build. Once complete, the package will be available on conda-forge.
+    
+To make changes or updates to your current package, you should make another pull request to your meta.yml file.
 Documentation
```
Comment (zhangcandrew) [inline docs/developer.rst]: Add a newline at line 165

File: docs/developer.rst [hunk 6]
```diff
@@ -144,7 +144,26 @@ Pull requests submitted to the repository will automatically be tested using
...
+    
+Open a pull request. Your package will be tested on Windows, Mac and Linux.
+    
+When your pull request is merged, a new repository called a feedstock will be created 
+in the github conda-forge organization, and your package will automatically 
+be uploaded and begin to build. Once complete, the package will be available on conda-forge.
+    
+
+To make changes or updates to your current package, you should make another pull request to your meta.yml file.
 Documentation
```
Comment (henrykironde) [inline docs/developer.rst]: new line between 166 and 167

**Comment by henrykironde** [comment]
@shreyneil, Thanks for working on this issue. Will get it in
